build-system/global-configs/canary-config.json

```diff
@@ -39,8 +39,8 @@
   "hidden-mutation-observer": 1,
   "ios-fixed-no-transfer": 1,
   "ios-scrollable-iframe": 0,
+  "macro-after-long-task": 1,
```

Comment:
do you want to turn this on in canary? otherwise, please revert and we only need a prod cherrypick

Reply:
oh noticed it's just a reorder